clang-tools-extra/clangd/ConfigCompile.cpp

```diff
@@ -535,7 +535,7 @@ struct FragmentCompiler {
     }
     if (Filters->empty())
       return std::nullopt;
-    auto Filter = [Filters](llvm::StringRef Path) {
+    auto Filter = [Filters = std::move(Filters)](llvm::StringRef Path) {
       for (auto &Regex : *Filters)
```

Comment:
```suggestion
      for (const auto &Regex : *Filters)
```

Same for both cases in other file.